dashboard/app/models/pd/application/application_base.rb

```diff
@@ -382,6 +382,18 @@ def default_response_score_hash
       {}
     end
 
+    def formatted_partner_contact_email
+      return nil unless regional_partner && regional_partner.contact_email_with_backup.present?
```

Comment:
remove the first conditional, use `regional_partner&.contact_email_with_backup.present?` instead